src/main/resources/data/minecraft/tags/worldgen/biome/has_structure/ruined_portal_nether.json

```diff
@@ -0,0 +1,9 @@
+{
+  "replace": false,
+  "values": [
+    "cinderscapes:ashy_shoals",
+    "cinderscapes:luminous_grove",
+    "cinderscapes:quartz_canyon",
+    "cinderscapes:blackstone_shales"
```

Comment:
These biomes are already included in the `minecraft:has_structure/ruined_portal_nether` tag because they are in the `minecraft:is_nether` tag.